src/app/shared/stackblitz/stackblitz-writer.ts

```diff
@@ -115,9 +115,13 @@ export class StackblitzWriter {
   }
 
   /** Reads the file and adds its text to the form */
-  _readFile(form: HTMLFormElement, data: ExampleData, filename: string, path: string): void {
+  _readFile(form: HTMLFormElement,
+            data: ExampleData,
+            filename: string,
+            path: string,
+            prependApp = true): void {
```

Comment:
Now would be a good time to add param descriptions, since `prependApp` is the first really non-obvious one

Reply:
Added comments. :)